corehq/apps/sms/handlers/fallback.py

```diff
@@ -23,6 +23,7 @@ def fallback_handler(verified_number, text, msg):
     inbound_meta = MessageMetadata(workflow=WORKFLOW_DEFAULT,
         messaging_subevent_id=inbound_subevent.pk)
     add_msg_tags(msg, inbound_meta)
+    msg.save()
```

Comment:
Was it intentional to not save this earlier, or was it just an oversight?

Reply:
What is the reason for this change and o you know the implications of it? I am a little concerned about possible side effects adding a call like this deep in the messaging stack that didn't otherwise exist